pkg/acsengine/const.go

```diff
@@ -90,6 +90,7 @@ var KubeImages = map[string]map[string]string{
 		"addonmanager":    "kube-addon-manager-amd64:v6.4-beta.2",
 		"dnsmasq":         "k8s-dns-dnsmasq-amd64:1.14.4",
 		"pause":           "pause-amd64:3.0",
+		"tiller":          "tiller:v2.5.1",
```

Comment:
Yeah, the idea behind the version-specific dictionaries are to carry version-specific config values for the various params. Based on 2.5.1 being present in all k8s versions, I would assume that that value would evolve universally; so, for example, if we decide in the future to bake a `v2.6.0` into acs-engine-curated clusters, we'd want to change the tiller version string value across all versions. (And thus the advantage to defining it one place as a constant.)

And yes, if we evolve tiller versions going forward only for newer versions of k8s, then we'd revisit how exactly to decompose this collection of string variables.

Hope that makes sense!